src/nvim/auevents.lua

```diff
@@ -154,6 +155,7 @@ return {
     LspAttach=true,
     LspDetach=true,
     LspRequest=true,
+    LspProgressUpdate=true,
```

Comment:
probably don't need "Update" in this one (if any).

```suggestion
    LspProgress=true,
```